datafusion/physical-plan/src/aggregates/mod.rs

```diff
@@ -2155,7 +2155,7 @@ mod tests {
         spill: bool,
     ) -> Result<()> {
         let task_ctx = if spill {
-            new_spill_ctx(2, 2812)
+            new_spill_ctx(2, 2886)
```

Comment:
Is this related?

Reply:
It is, with original value it will panic with `ResourcesExhausted` in the merge phase, so I have increase it. Did not do in depth analysis of the problem